
---
float64 goal_dist
float64 velm_x
float64 velm_z
